### DISCUSSION TITLE
Use translated fields in display - how?

### DISCUSSION THREAD

**Discussion by danieljwestman** [directus/directus#5464] [Archive]
Bug or not, but this behavior seems a bit off. 

I want to display one of the translated fields in an item but can't figure out how... "Translations" is not expandable like the other relations...

<img width="974" alt="Skärmavbild 2021-05-04 kl  16 24 33" src="https://user-images.githubusercontent.com/20950568/117067359-34bd8780-ad2a-11eb-8503-2f1b93839949.png">

**Comment by danieljwestman** [comment]
@rijkvanzanten is quicker than Lucky Luke 😂

**Reply by rijkvanzanten** [reply]
Pow! 💥 🔫  🤠

**Comment by rijkvanzanten** [comment]
`translations` is a o2m style interface here, so you can't just show any nested item of the relationship inside the row of the m2m. That being said, you should be able to setup the `related-values` display for the translations o2m itself, and use that to control what's rendered in the preview (It'll say `x Items` with a click-popover with item previews). We're aiming to add a dedicated `translations` "preview" display that will render the item in the currently used language, but that's coming later

**Reply by danieljwestman** [reply]
Okay, IC.

> We're aiming to add a dedicated translations "preview" display that will render the item in the currently used language

💯
